src/InputState.cpp

```diff
@@ -66,7 +66,7 @@ InputState::InputState(void)
 	, current_touch()
 	, dump_event(false)
 	, file_version(new Version())
-	, file_version_min(new Version(1, 6, 28))
+	, file_version_min(new Version(1, 6, 29))
```

Comment:
This version should reflect the oldest engine version that the `keybindings.txt` file is compatible with. So please:

- In `Version.cpp`, change `VersionInfo::ENGINE` to `1, 9, 20`
- Change the `file_version_min` in the line highlighted here to `1, 9, 20`

Other than that, this commit looks good. I tested with a few layouts and everything worked.